build/run_unit_group.sh

```diff
@@ -129,7 +129,7 @@ function test_group_proxy() {
 
 function test_group_other() {
   mvn_test --clean --install \
-           -pl '!org.apache.pulsar:distribution,!org.apache.pulsar:pulsar-offloader-distribution,!org.apache.pulsar:pulsar-server-distribution,!org.apache.pulsar:pulsar-io-distribution' \
+           -pl '!org.apache.pulsar:distribution,!org.apache.pulsar:pulsar-offloader-distribution,!org.apache.pulsar:pulsar-server-distribution,!org.apache.pulsar:pulsar-io-distribution,!org.apache.pulsar:pulsar-all-docker-image' \
```

Comment:
Does this change related to this PR?

Reply:
<img width="1273" alt="image" src="https://user-images.githubusercontent.com/10069311/192741853-796e20f6-c74f-406d-94d0-a1607f7e8521.png">


If we don't exclude `org.apache.pulsar:pulsar-all-docker-image`, then there will appear this error.

Reply:
Although I think it's something related to network, it does no harm to exclude `org.apache.pulsar:pulsar-all-docker-image` in `test_group_other`.